### ISSUE TITLE
TMM: Display different rating / league icon + name for each queue

### ISSUE THREAD

**Issue by Askaholic** [FAForever/downlords-faf-client#1742]
Right now TMM games will display player's ladder ratings. For now we probably don't want to pass any rating values to ForgedAlliance.exe during TMM games since we haven't really decided how to display rating (possible league system). 

Games should only have ladder1v1 rating passed in if they are actually ladder1v1 games (use the ladder1v1 gamemode).

**Wanna have the bug fixed quickly?**
Visit Issue hunt...
[![Issue hunt](https://github.com/BoostIO/issuehunt-materials/raw/master/v1/issuehunt-button-v1.svg?sanitize=true)](https://issuehunt.io/r/FAForever/downlords-faf-client)

**Comment by cleborys** [comment]
As far as I can see, when launching a game you first patch the gamefiles according to the featured mod and then start the exe with all other arguments passed in.


mod patching step:
https://github.com/FAForever/downlords-faf-client/blob/a055a57eee24ba729715ecc1dfac137cd7252699/src/main/java/com/faforever/client/game/GameService.java#L500-L501
a step in between to start the ladder search:
https://github.com/FAForever/downlords-faf-client/blob/a055a57eee24ba729715ecc1dfac137cd7252699/src/main/java/com/faforever/client/game/GameService.java#L502
argument passing step:
https://github.com/FAForever/downlords-faf-client/blob/a055a57eee24ba729715ecc1dfac137cd7252699/src/main/java/com/faforever/client/game/GameService.java#L504-L511


In the second step the `RatingMode` decides which rating will be displayed.
Currently there are only two places going through this patch(->search)->start flow, one for custom games and the one above which sets the mod, starts a ladder search, and then launches and I assume the second one was used to start tmm games in testing?

Long story short I don't think this can be worked on without deciding how games are started from the tmm queues, which probably needs a different flow like search->patch->start instead of patch->search->start.
That's not yet done in `feature/team-matchmaking`, is it?

**Comment by Askaholic** [comment]
Why would it be different for TMM? I think you would still want to patch before searching.

**Comment by cleborys** [comment]
I was assuming you'd possibly queue up for multiple queues at the same time and that different queues would have different featured mods to patch with, but you don't know which one until you receive a `match_found` or similar.

**Comment by Askaholic** [comment]
Yea, but I don't think they can ever conflict with each other since there is no way to specify a featured mod version other than the latest version.

**Comment by Askaholic** [comment]
I think we'll have to do #1777 first, so maybe this isn't so beginner friendly after all.

**Comment by cleborys** [comment]
Thats pretty much what I was trying to say :) :+1:

**Comment by Geosearchef** [comment]
> I was assuming you'd possibly queue up for multiple queues at the same time and that different queues would have different featured mods to patch with, but you don't know which one until you receive a `match_found` or similar.

There's no way to do that, we should just display the best rating (from a chosen selection) on the party members and display a rating for each queue individually below the queue.

**Comment by Askaholic** [comment]
Maybe we don't want to display a rating at all for matchmaker games, since we are trying to move towards a league system anyway. Would be cool to display a league avatar I guess, but that would have to be added to the lua code.

**Comment by Sheikah45** [comment]
This has been resolved and the proper rating is displayed now since the tmm client release
